css/main2.css

```diff
@@ -111,6 +117,15 @@ H2.textboxtitle {
     margin-bottom: 0px;
 }
 
+DIV.column {
+    float: left;
+    margin-right: 20px;
+}
+
+DIV.column_container {
```

Comment:
It looks like you're trying to use the whole `column_container` and `column` to effect a table without actually creating a table. While this works just fine on Firefox and Edge, it doesn't appear to work with Chrome. (You really need to test with all three...and probably Safari, as well.) Would it be better to just create a table and lay things out properly?

Also, the `Buy me a coffee` button should be vertically aligned with the social media icons.

Finally, I think the _Donate_ section should be right-aligned on the page.

Reply:
The column thing is partially resolved by my Chrome when I use an explicit IP address instead of `localhost`. I'll have to investigate that. My other comments here are still valid.

Reply:
I usually find using table for layout soon becomes unworkable, except for tabular data, but divs are indeed overengineering in this case.  Time to simplify.